src/ipc/totp-get.test.ts

```diff
@@ -0,0 +1,35 @@
+import { fakeIpc } from '../../test/ipc'
+import mockOf from '../../test/mockOf'
+import exec from '../utils/exec'
+import register, { channel as totpGetChannel } from './totp-get'
+
+const execMock = mockOf(exec)
+jest.mock('../utils/exec')
+
+beforeEach(() => {
+  execMock.mockReset()
+  execMock.mockResolvedValue({ stdout: '', stderr: '' })
+})
+
+const { ipcMain, ipcRenderer } = fakeIpc()
+register(ipcMain)
+
+test('call to totp calls appropriate shell command and returns the right data', async () => {
```

Comment:
Are you also gonna mock the case where tpm2-totp hasn't been initialized yet?

Reply:
yeah, happy to, can you tell me what happens then? :)

Reply:
it'll throw an error, so you'll get input on stderr but not on stdout (I think).